packages/fxa-content-server/app/tests/spec/views/mixins/service-mixin.js

```diff
@@ -63,6 +63,12 @@ describe('views/mixins/service-mixin', () => {
       assert.equal(context.get('service'), 'sync');
       assert.equal(context.get('serviceName'), 'Firefox Sync');
     });
+
+    it('sets `serviceName` default from the relier', () => {
+      const context = new Backbone.Model({});
+      view.setInitialContext(context);
+      assert.equal(context.get('serviceName'), 'Account Settings');
```

Comment:
Ahha! I forgot, we have to ensure the service name gets translated, so in the service-mixin when setting the [context's serviceName](https://github.com/mozilla/fxa/blob/3d0a657355337c27c847bc511fa4560658e17302/packages/fxa-content-server/app/scripts/views/mixins/service-mixin.js#L17), run it through `this.translate`

Reply:
I made this change (haven't pushed due to my other comment), and I knew about `t('text')` for L10n but was unaware I needed `this.translate` as well. I was going to ask if we have this in the docs but [I found it](https://mozilla.github.io/ecosystem-platform/docs/fxa-engineering/fxa-localization)! 😏

I see "`t` is a signal to the l10n extraction script that the embedded string is meant for translation". Is `translate` (looks like it comes from `base.js`) just to protect from unescaped HTML?

Reply:
`t` as a function in JS doesn't actually do any translation, it's just the signal to extract the string. `{{#t}}` and `{{#unsafeTranslate}}` within the mustache templates translates, but everything else needs translated manually. I should add that to the content server gotchas.

Reply:
Hey @rfk, I'm sorry, I totally missed the notification that you commented on this a few days ago.

Were you able to figure out what you needed? I'm still a little fuzzy on our localization process and asked Shane a couple of questions here about it. It does seem like `this.translate('serviceName')` may be incorrect, does the following seem right?

```
const relierContext = this.relier.pick('service', 'serviceName');
relierContext.serviceName = this.translate(relierContext.serviceName);
context.set(relierContext);
```